docs/models.rst

```diff
@@ -140,9 +206,14 @@ The ``Meta`` class
 In event model we got some more fields, that could be interesting for us.
 
 ``fields.ForeignKeyField('models.Tournament', related_name='events')``
-    Here we create foreign key reference to tournament. We create it by referring to model by it's literal, consisting of app name and model name. `models` is default app name, but you can change it in `class Meta` with `app = 'other'`.
+    Here we create foreign key reference to tournament. We create it by
+    referring to model by it's literal, consisting of app name and model
+    name. `models` is default app name, but you can change it in `class Meta`
+    with `app = 'other'`.
```

Comment:
Too much churn, if this was an automated editor, see if you can turn off the hard wrapping? We run the risk of breaking things without noticing.

Reply:
Ok